packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.service.js

```diff
@@ -17,12 +21,25 @@ export default class IpAgoraOrder {
       .catch(() => null);
   }
 
+  handleErrorOrServices({ errors, results }) {
+    const filteredErrors = errors.filter(({ msg }) => {
+      const [errorCode] = msg.match(/\d+/);
+      return ![400, 404].includes(parseInt(errorCode, 10));
+    });
+    if (isArray(filteredErrors) && !isEmpty(filteredErrors)) {
```

Comment:
Try avoid using `lodash` when it's possible
```suggestion
    if (filteredErrors?.length) {
```